lib/widget/camera.dart

```diff
@@ -54,18 +96,33 @@ class _CameraScreenState extends State<CameraScreen> {
   @override
   void initState() {
     super.initState();
-    errorString = 'You just pick at least ${widget.maxCount} image';
+    WidgetsBinding.instance.addObserver(this);
+    errorString = 'You just pick at least ${widget._controller.maxCount} image';
     initCamera().then((_) {
       ///initialize camera and choose the back camera as the initial camera in use.
       if (cameras.length == 2) {
-        if (widget.initialCamera == 'back') {
-          setCamera(0);
-        } else {
+        if (widget._controller.initialCamera == 'back') {
           setCamera(1);
```

Comment:
Does the library not having an enum like FRONT / BACK? How do we know camera1 is the back and camera0 is the front?

Reply:
There’s no enum defined in library. I have checked, whenever front camera opens at 0 index and back at 1 index, camera name’s got written in camera description but there’s no description about front or back camera

Reply:
yes i done my changes and using enum and remove 0 or 1